Sources/RInAppMessaging/Extensions/UILabel+IAM.swift

```diff
@@ -14,7 +14,7 @@ internal extension UILabel {
         paragraphStyle.lineSpacing = lineSpacing
 
         let attributedString: NSMutableAttributedString
-        if let attributedText = self.attributedText {
+        if let attributedText, attributedText.length > 0 {
```

Comment:
Adding new condition because attributedText will never be nil.
source: [link](https://developer.apple.com/forums/thread/118581)

Reply:
`attributedText` would be nil if `UILabel().text` is nil. Since already `guard else` check is there at the top(line 9), this `if` condition will pass all the time in the both old and new approaches. Better, have a look and remove `else` part.

Reply:
When I'm creating an instance of `Label()` I get `nil` for `attributedText`. It's true that if you set `text` to some non-nil value, you'll also get a non-nil value from `attributedText`. Setting `text` to nil also sets `attributedText` to nil. So I don't see any reason to add that additional condition.

Reply:
Oh ok. it's because of the guard statement in line 9. So I guess, because `attributedText` value is 'synchronized' with `text` value, you can replace the guard condition to `guard let attributedText = self.attributedText` and then proceed without any additional conditions and always use `NSMutableAttributedString(attributedString: attributedText)`

Reply:
And also I think we don't have to restrict this function to non-empty texts only